mlflow/_build_backend/backend.py

```diff
@@ -0,0 +1,27 @@
+# https://setuptools.pypa.io/en/latest/build_meta.html#dynamic-build-dependencies-and-other-build-meta-tweaks
+import subprocess
+import sys
+from pathlib import Path
+
+from setuptools import build_meta as _orig_build_meta
+from setuptools.build_meta import *  # noqa: F403
+
+
+def _update_sha():
+    sys.stdout.write("Updating REF in mlflow/ref.py...\n")
+    sys.stdout.write(f"Current directory: {Path.cwd()}\n")
+    try:
+        sha = subprocess.check_output(["git", "rev-parse", "HEAD"]).decode("utf-8").strip()
```

Comment:
Is git always available?

Reply:
Good question. No, that's the reason why it's in try-catch.